frontend-student/src/components/navBar/navItems.jsx

```
@@ -1,38 +1,44 @@
-        <Link to="/">Home</Link>
+        <Link to='/'>Home</Link>
       </button>
 
       <button
         className={`${
-          location.pathname === "/courses" ? "bg-blue-600 text-white" : "bg-transparent text-blue-600"
+          location.pathname === '/courses'
+            ? 'bg-blue-600 text-white'
+            : 'bg-transparent text-blue-600'
         } py-2 px-4 rounded-md border border-blue-600 hover:bg-blue-600 hover:text-white transition duration-200`}
       >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)